packages/blockchain-link/src/workers/blockbook/websocket.ts

```diff
@@ -187,8 +187,14 @@ export default class Socket extends EventEmitter {
         // create deferred promise
         const dfd = createDeferred<void>(-1);
 
-        // initialize connection
-        const ws = new WebSocket(url);
+        // initialize connection,
+        // options are not used in web builds (see ./src/utils/ws)
+        const ws = new WebSocket(url, {
+            headers: {
+                Origin: 'https://node.trezor.io',
+                'User-Agent': 'Mozilla', // user-agent is essential for blockbook servers, doesn't have to be valid tho
```

Comment:
Let's set this to Trezor Suite

```suggestion
                'User-Agent': 'Trezor Suite',
```

Reply:
a19663b